feature_apps/src/main/java/cm/aptoide/pt/feature_apps/data/App.kt

```diff
@@ -210,24 +209,3 @@ val randomApp
     ),
     developerName = getRandomString(range = 2..5, capitalize = true)
   )
-
-private fun getRandomString(
```

Comment:
I'm glad you also came up with the idea to extract this to `extensions` :-)
Though I already extracted this in my PR. Lets merge that first and then rebase this PR on top.